.github/workflows/tribits_testing.yml

```diff
@@ -0,0 +1,70 @@
+name: TriBITS Testing
+
+# Controls when the action will run.
+# Triggers the workflow on push or pull request
+# for the master and develop branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    strategy:
+      max-parallel: 1
```

Comment:
@marcinwrobel1986, just curious, but why is `max-parallel` set to just `1`?  Why not run all of these jobs at the same time?  Why not let them run all at the same time?    From looking at:

* https://docs.github.com/en/actions/reference/usage-limits-billing-and-administration

it looks like it can be as high as 20 for a free plan.

Reply:
@bartlettroscoe I set it to one, as I was getting strange cancellation of jobs E.g. when installing ubuntu packages. Once you are sure all jobs are succeded with 1 job at the time, please feel free to change it and check if there is no errors. I think it has something to do with resources. It could be as high as 20 parallel jobs, but if the resources are limited (CPU, RAM) it will not be working correctly, but I would give it a try anyway, but after making sure all jobs are passing.